src/vs/base/browser/keyboardEvent.ts

```diff
@@ -114,6 +114,13 @@ export function printStandardKeyboardEvent(e: StandardKeyboardEvent): string {
 	return `modifiers: [${modifiers.join(',')}], code: ${e.code}, keyCode: ${e.keyCode} ('${KeyCodeUtils.toString(e.keyCode)}')`;
 }
 
+/**
```

Comment:
Note that we have similar code already that we should revisit to use this method or align:

https://github.com/microsoft/vscode/blob/1332311cbaf9f21fb0a11ce5cfd9216208c52018/src/vs/base/browser/dom.ts#L1815-L1817